lib/widgets/content_list_view/song_tile.dart

```diff
@@ -161,6 +186,12 @@ class SongTile extends SelectableWidget<SelectionEntry> {
 
 class _SongTileState extends SelectableState<SelectionEntry<Song>, SongTile> with ContentTileComponentsMixin {
   Color? previousBackgroundColor;
+  /// The padding that is added to the top of the subtitle widget.
+  static const subtitleTopPadding = 4.0;
+  /// The padding that is added to the bottom of the subtitle widget.
+  static const subtitleBottomPadding = 3.0;
+  /// The [TextStyle] used for the title text from the [theme].
+  static TextStyle titleTheme(ThemeData theme) => theme.textTheme.headline6!;
```

Comment:
I'd prefer these to be defined privately in global scope at the top of the file, like other constants

Reply:
Done: c183ede651696b4e3b449316f0ef9d8f01ae9889